### ISSUE TITLE
Interpreting output as a docked protein complex

### ISSUE THREAD

**Issue by cshih14** [ketatam/DiffDock-PP#17]
I was able to fix the output PDB files according to #10 to get visuals that look more like proteins.  However, when combining the ligand and receptor PDBs, the ligand and receptor appear to have overlapping coordinates as if they are just smashed together rather than docked as a complex.  Here is a screenshot of the combined PDB visualized:

![Screenshot 2023-07-31 at 8 47 18 AM](https://github.com/ketatam/DiffDock-PP/assets/54996682/e560a597-beef-4c77-b438-1557917505ec)

In the `visualization/` output directory I get ligand PDB files with numbers appended 0 through 40, plus a ‘-gt' appended ligand PDB and a receptor PDB. Still trying to figure out the meaning of these outputs.

Is combining the ligand PDB appended with ‘-40’ (final timestep?) with the receptor output PDB into one single "complex" PDB the correct way to get the output complex?

**Comment by YFeriel** [comment]
Hi @cshih14 @ketatam ,

Did you find a solution? I'm facing a similar issue and would appreciate any insights.

Thanks!

**Comment by noah-c-noah-do** [comment]
If be interested in any updates/info as well. Currently I only get the single output structure regardless of NUM_SAMPLES I set.

I thought the pickle would let me creat the 40 output structures but I can't see any difference between them except the last number as described in another issue. 

The visualization structures seem (as far as I can tell) to be the steps in the diffusional process and unrelated to the docked outputs.

@ketatam any chance you could provide some clarity or updates? I'm starting to worry this software has maybe been abandoned...

**Comment by Sue-Fwl** [comment]
> If be interested in any updates/info as well. Currently I only get the single output structure regardless of NUM_SAMPLES I set.
> 
> I thought the pickle would let me creat the 40 output structures but I can't see any difference between them except the last number as described in another issue.
> 
> The visualization structures seem (as far as I can tell) to be the steps in the diffusional process and unrelated to the docked outputs.
> 
> @ketatam any chance you could provide some clarity or updates? I'm starting to worry this software has maybe been abandoned...

Hello @noah-c-noah-do,
Any news on how to work with the pickled 40+ HetroData objects ?

**Comment by noah-c-noah-do** [comment]
> > If be interested in any updates/info as well. Currently I only get the single output structure regardless of NUM_SAMPLES I set.
> 
> > 
> 
> > I thought the pickle would let me creat the 40 output structures but I can't see any difference between them except the last number as described in another issue.
> 
> > 
> 
> > The visualization structures seem (as far as I can tell) to be the steps in the diffusional process and unrelated to the docked outputs.
> 
> > 
> 
> > @ketatam any chance you could provide some clarity or updates? I'm starting to worry this software has maybe been abandoned...
> 
> 
> 
> Hello @noah-c-noah-do,
> 
> Any news on how to work with the pickled 40+ HetroData objects ?
> 
> 

Nope, never got a response and couldn't spend the time to figure out what was the issue.

**Comment by Sue-Fwl** [comment]
Appreciate the quick reply, @noah-c-noah-do !
Guess I'll stop at this note as I've spent enough time trying to figure the output out without any clue.
